share/spack/gitlab/cloud_pipelines/stacks/developer_tools/spack.yaml

```diff
@@ -0,0 +1,65 @@
+spack:
+  view: false
+  packages:
+    all:
+      require: target=x86_64_v3
+  concretizer:
+    reuse: true
+    unify: true
+  definitions:
+    - default_specs:
+      # editors
+      - neovim~no_luajit
+      - py-pynvim
+      - emacs@29.1+json+native+treesitter # note, pulls in gcc
```

Comment:
Just personal curiosity, is `emacs+native` noticeably faster? Haven't used it before on/can't use it on MacOS.

Reply:
It depends, but it can be substantial.  I use it on macos and linux with doom emacs as a mail client option, and it's significantly faster.

Reply:
Oh cool! If `emacs+native` brings in `gcc` as a dependency how are you able to build on macOS? Last I remember compiling gcc from apple-clang had a bug that was failing builds.

Reply:
I'm not currently building it through spack, and apparently not currently running the version I thought I was.  For quite a while I was running the homebrew package from this tap: railwaycat/emacsmacport with the native compilation option.  Will have to try the spack version.